src/engines/html5/html5.js

```diff
@@ -111,11 +162,11 @@ export default class Html5 extends FakeEventTarget implements IEngine {
    * @returns {void}
    */
   attach(): void {
-    for (let playerEvent in Html5Events) {
-      this._eventManager.listen(this._el, Html5Events[playerEvent], () => {
-        this.dispatchEvent(new FakeEvent(Html5Events[playerEvent]));
+    Object.keys(Html5Events).forEach((html5Event) => {
```

Comment:
I hate `for in` since eslint shout on you if u don't include the `hasOwnProperty` protection.
This is seems like two actions added, but `for in` do exactly the same behind the scene.

BTW, u can run the tests here:
https://jsperf.com/object-keys-vs-for-in-with-closure/3
and see the `Object.keys().forEach()` is **much more** faster.